allennlp/commands/make_vocab.py

```diff
@@ -70,8 +72,8 @@ def make_vocab_from_params(params: Params):
     prepare_environment(params)
 
     vocab_params = params.pop("vocabulary", {})
-    vocab_dir = vocab_params.get('directory_path')
-    if vocab_dir is None:
+    vocab_dir = vocab_params.pop('directory_path', {})
```

Comment:
one last nit: can you change the default value to `None`? (`{}` doesn't make sense as a possible value)